examples/periodic_snapshotter.cpp

```diff
@@ -50,59 +60,95 @@
 namespace laser_assembler
 {
 
-class PeriodicSnapshotter
+class PeriodicSnapshotter: public rclcpp::Node
 {
 
 public:
 
-  PeriodicSnapshotter()
+  explicit PeriodicSnapshotter(const std::string & service_name)
+  : Node(service_name)
   {
-    // Create a publisher for the clouds that we assemble
-    pub_ = n_.advertise<sensor_msgs::PointCloud> ("assembled_cloud", 1);
-
-    // Create the service client for calling the assembler
-    client_ = n_.serviceClient<AssembleScans>("assemble_scans");
+    std::cerr<< " Inside PeriodicSnapshotter constructor " << "\n";
 
-    // Start the timer that will trigger the processing loop (timerCallback)
-    timer_ = n_.createTimer(ros::Duration(5,0), &PeriodicSnapshotter::timerCallback, this);
+    client_ = this->create_client<laser_assembler_srv_gen::srv::AssembleScans>("build_cloud");
+    
+    if (!client_->wait_for_service(20s)) {
+      printf("Service not available after waiting");
+      return;
+    }
+    printf(" Service assemble_scans started successfully");
+    //ROS_INFO("Service [%s] detected", SERVICE_NAME.c_str());
+    
+    // Create a function for when messages are to be sent.
+    auto timer_callback = [this]() -> void {
+        // msg_->name = "Hello World: " + std::to_string(count_++);
```

Comment:
Please remove all the commented out code and the "Hello world" stuff.

Reply:
Removed commented out code in next commits.